bot.rb

```diff
@@ -241,5 +262,38 @@ def process_code_signing(issue)
     def smart_sleep
       sleep 5
     end
+
+    # Create a hash of PR numbers (as strings) to release tag names indicating in which release
+    # a given PR was mentioned in the release notes. For example:
+    #
+    # {
+    #   "8594"=>"2.22.0",
+    #   "8592"=>"2.22.0",
+    #   "8595"=>"2.22.0",
+    #   "8593"=>"2.21.0",
+    #   "8596"=>"2.21.0",
+    #   "8564"=>"2.20.0"
+    # }
+    def map_prs_to_releases(releases)
+      prs_to_releases = {}
+      releases = releases.select { |r| !r.draft && !r.prerelease }
```

Comment:
Should this be an `||` check rather than `&&`? This is checking for if the release is a prerelease draft, do we want to put the comment if the PR is in a release draft or in a prerelease just not both?